packages/config/config/devices/0x0403/plha10000.json

```
@@ -0,0 +1,100 @@
+{
+	"manufacturer": "ABUS Security-Center GmbH & Co. KG",
+	"manufacturerId": "0x0403",
+	"label": "PLHA10000",
+	"description": "Z-Wave 230V relay module",
```

This should be Title Case
```suggestion
	"description": "Z-Wave 230V Relay Module",
```